client/app/assets/less/inc/base.less

```
@@ -89,7 +87,7 @@ strong {
 .admin-page-layout {
   .container {
     width: 100%;
-    max-width: 1200px;
+    max-width: none;
```

```suggestion
    max-width: 1600px;
```

How about we just increase the limit? It can be a bit weird for large screens.

---

it does looks weird. can we play with the placement/size to make it looks better?

---

I tried `1600px` on my 1920*1080 screen, but liked no limit variant more. But we can vote :-)

---

Idk, I think I'd rather go with an even larger max-width than with `none`, with `none` we may have to start introducing max-width in some other elements (e.g.: the sidebar for list pages).